vips/vip-400/bscmainnet.ts

```diff
@@ -0,0 +1,334 @@
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            WEETH,
+            [WEETH_ONEJUMP_REDSTONE_ORACLE, WEETH_ONEJUMP_CHAINLINK_ORACLE, WEETH_ONEJUMP_CHAINLINK_ORACLE],
+            [true, true, true],
+          ],
+        ],
+      },
+
+      // Swap router
+      {
+        target: SWAP_ROUTER,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+
+      // Liquid Staked ETH pool
+      {
+        target: COMPTROLLER,
+        signature: "acceptOwnership()",
+        params: [],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "setPriceOracle(address)",
+        params: [RESILIENT_ORACLE],
+      },
+      {
+        target: POOL_REGISTRY,
+        signature: "addPool(string,address,uint256,uint256,uint256)",
+        params: ["Liquid Staked ETH", COMPTROLLER, CLOSE_FACTOR, LIQUIDATION_INCENTIVE, MIN_LIQUIDATABLE_COLLATERAL],
+      },
+      {
+        target: COMPTROLLER,
+        signature: "setPrimeToken(address)",
+        params: [PRIME],
+      },
+
+      // Markets
+      ...Object.values(newMarkets).flatMap(({ vToken, initialSupply, riskParameters }) => [
+        {
+          target: vToken.address,
+          signature: "setReduceReservesBlockDelta(uint256)",
+          params: ["28800"],
+        },
+        {
+          target: vToken.address,
+          signature: "setReserveFactor(uint256)",
+          params: [riskParameters.reserveFactor],
+        },
+        {
+          target: TREASURY,
+          signature: "withdrawTreasuryBEP20(address,uint256,address)",
+          params: [vToken.underlying.address, initialSupply.amount, NORMAL_TIMELOCK],
+        },
+        {
+          target: vToken.underlying.address,
+          signature: "approve(address,uint256)",
+          params: [POOL_REGISTRY, initialSupply.amount],
+        },
+        {
+          target: POOL_REGISTRY,
+          signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+          params: [
+            [
+              vToken.address,
+              riskParameters.collateralFactor,
+              riskParameters.liquidationThreshold,
+              initialSupply.amount,
+              initialSupply.vTokenReceiver,
+              riskParameters.supplyCap,
+              riskParameters.borrowCap,
+            ],
+          ],
+        },
+        {
+          target: vToken.underlying.address,
+          signature: "approve(address,uint256)",
+          params: [POOL_REGISTRY, 0],
+        },
+        {
+          target: vToken.address,
+          signature: "setProtocolSeizeShare(uint256)",
+          params: [riskParameters.protocolSeizeShare],
+        },
+      ]),
+
+      // Conversions Config
+      ...Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => {
+        const tokens = [WSTETH, WEETH];
+        const conversionConfigs = tokens.map(() => [0, ConversionAccessibility.ALL]);
```

Comment:
I would set 1e14 for the conversion incentives here, as it was made in the [VIP-350](https://app.venus.io/#/governance/proposal/350?chainId=56) for the rest of the markets on BNB Chain, following the Chaos labs recommendations

Reply:
https://github.com/VenusProtocol/vips/pull/373/commits/0adbbd1173b83e29af85e2d7d5a00373a047682e